lilach-client/src/main/resources/org/cshaifasweng/winter/Customer_Cart.fxml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
+
+<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" prefHeight="510.0" prefWidth="904.0" style="-fx-background-color: #FFFFFF;" xmlns="http://javafx.com/javafx/10.0.2-internal" xmlns:fx="http://javafx.com/fxml/1" fx:controller="org.cshaifasweng.winter.CartController">
+    <children>
+        <TextField layoutY="20.0" prefHeight="77.0" prefWidth="904.0" text="My Shopping Cart">
+            <font>
+                <Font size="36.0" />
+            </font>
+            <opaqueInsets>
+                <Insets top="1.0" />
+            </opaqueInsets></TextField>
+      <Button layoutX="670.0" layoutY="348.0" mnemonicParsing="false" onAction="#GoToOrder" prefHeight="50.0" prefWidth="104.0" text="Check Out">
```

Comment:
Once again, pay attention to the first letter - should not be capital.

Reply:
i will decide about cart model today.
will change to  goToOrder